validator/test/fixtures/gke_node_auto_repair/constraints/enable_auto_repair/data.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKENodeAutoRepairConstraintV1
+metadata:
+  name: enable_auto_repair
+  annotations:
+    description: Ensure automatic node repair is enabled on all node pools in a GKE cluster
+    benchmark: CIS11_7.07
```

Comment:
The benchmark annotation style has been updated, can you also update this and the rest (& previous) accordingly?

Example of the new style:
https://github.com/forseti-security/policy-library/pull/137/files

Reply:
@joecheuk updated this on my open PRs. by "& previous" do you mean my previously merged policies or do you mean all of the policies currently in the repo?

Reply:
I meant the policies you authored.

Reply:
Just to be sure, can you also add in the comment
```
# This constraint is not certified by CIS.
```
That's a requirement from the CIS side.